autoRSA.py

```diff
@@ -572,6 +577,11 @@ async def rsaadd(ctx, broker=None, credentials=None):
                         raise Exception(
                             "Invalid credentials. Just enter your email for Fennel."
                         )
+                elif broker =="Dspac":
+                    if "@" not in credentials or len(credentials.split(":")) != 1:

```

Comment:
@mono300genuine it's supposed to be username not email

Reply:
resloved